Tasks/Src/Task_Init.c

```diff
@@ -23,14 +24,23 @@ void Task_Init(void *p_arg) {
 
     RTOS_BPS_MutexCreate(&WDog_Mutex, "Watchdog Mutex");
 
+    // 1
+    RTOS_BPS_TaskCreate(&CheckContactor_TCB,    // TCB
+				"Task_CheckContactor",          // Task Name (String)
+				Task_CheckContactor,            // Task function pointer
+				(void *)0,                      // Task function args
+				TASK_CHECK_CONTACTOR_PRIO,      // Priority
+				CheckContactor_Stk,             // Stack
+				TASK_CHECK_CONTACTOR_STACK_SIZE
+                );
     //2
-    RTOS_BPS_TaskCreate(&CriticalState_TCB,	    // TCB
-            "TASK_CRITICAL_STATE",	            // Task Name (String)
-            Task_CriticalState,				    // Task function pointer
-            (void *)0,				            // Task function args
-            TASK_CRITICAL_STATE_PRIO,		    // Priority
-            CriticalState_Stk,				    // Stack
-            TASK_CRITICAL_STATE_STACK_SIZE);
+    // RTOS_BPS_TaskCreate(&CriticalState_TCB,	    // TCB
```

Comment:
Why is the critical state replaced by checkcontactor task? Shouldn't we keep both